app/src/store/reducers/DatasetsReducer.ts

```diff
@@ -11,7 +11,7 @@ const initialState: DatasetsStore = {
   selectedDataset: null,
 };
 
-export const doesDatasetMatchesTags = (
+export const doesDatasetMatchTags = (
```

Comment:
It is not a concern of the reducer. Please move the function to the utils area.

Reply:
I actually just renamed the function here - it's preexisting ;)

Reply:
But yeah - I moved it now :)